airflow/providers/google/cloud/operators/dataform.py

```diff
@@ -348,6 +349,81 @@ def execute(self, context: Context):
         return WorkflowInvocation.to_dict(result)
 
 
+class DataformQueryWorkflowInvocationOperator(GoogleCloudBaseOperator):
```

Comment:
I have changed it to `DataformQueryWorkflowInvocationActionsOperator` as the official doc already contains class with as ListWorkflowInvocations. In case we add an operator for this class in future, it might create ambiguity for the user.